### ISSUE TITLE
`nixpkgs.python35Packages.pgcli` can not be installed

### ISSUE THREAD

**Issue by phanimahesh** [NixOS/nixpkgs#16945]
## Issue description
`nixpkgs.python35Packages.pgcli` can not be installed

### Steps to reproduce
```
  nox pgcli                                        127 ↵  
Refreshing cache
1 python2.7-pgcli-0.20.1 (nixpkgs.python27Packages.pgcli)
    Command-line interface for PostgreSQL
2 python3.5-pgcli-0.20.1 (nixpkgs.python35Packages.pgcli)
    Command-line interface for PostgreSQL
Packages to install: 2
installing ‘python3.5-pgcli-0.20.1’
error: while evaluating the attribute ‘propagatedNativeBuildInputs’ of the derivation ‘python3.5-pgcli-0.20.1’ at /nix/store/7nygbmc03nxm24iz59s6gwwh9j1q5ham-nixpkgs/nixpkgs/pkgs/development/python-modules/generic/default.nix:108:3:
while evaluating ‘makeOverridable’ at /nix/store/7nygbmc03nxm24iz59s6gwwh9j1q5ham-nixpkgs/nixpkgs/lib/customisation.nix:54:24, called from /nix/store/7nygbmc03nxm24iz59s6gwwh9j1q5ham-nixpkgs/nixpkgs/pkgs/top-level/python-packages.nix:16474:20:
while evaluating anonymous function at /nix/store/7nygbmc03nxm24iz59s6gwwh9j1q5ham-nixpkgs/nixpkgs/pkgs/development/python-modules/generic/default.nix:9:1, called from /nix/store/7nygbmc03nxm24iz59s6gwwh9j1q5ham-nixpkgs/nixpkgs/lib/customisation.nix:56:12:
prompt_toolkit-1.0.0 not supported for interpreter python3.5m
Traceback (most recent call last):
  File "/nix/store/m01qr34jsbfp81hdm8720wyr9hh5rnra-nox-0.0.2/bin/.nox-wrapped", line 12, in <module>
    sys.exit(main())
  File "/nix/store/wxafxxg9m2igxm89jm6hrvcq732c9ylv-python3.4-click-6.6/lib/python3.4/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/nix/store/wxafxxg9m2igxm89jm6hrvcq732c9ylv-python3.4-click-6.6/lib/python3.4/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/nix/store/wxafxxg9m2igxm89jm6hrvcq732c9ylv-python3.4-click-6.6/lib/python3.4/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nix/store/wxafxxg9m2igxm89jm6hrvcq732c9ylv-python3.4-click-6.6/lib/python3.4/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/nix/store/m01qr34jsbfp81hdm8720wyr9hh5rnra-nox-0.0.2/lib/python3.4/site-packages/nox/search.py", line 83, in main
    subprocess.check_call(['nix-env', '-iA', '--show-trace'] + attributes)
  File "/nix/store/aqyvrkns2ykvb83pmmbb7ban3kwgf0n9-python3-3.4.4/lib/python3.4/subprocess.py", line 561, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['nix-env', '-iA', '--show-trace', 'nixpkgs.python35Packages.pgcli']' returned non-zero exit status 1
```

## Technical details

* System:
```
$ lsb_release -a                                                                                                         1 ↵  
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04 LTS
Release:	16.04
Codename:	xenial
```
* Nix version:
```
$ nix-env --version                                                                                                           
nix-env (Nix) 1.11.2
```
* Nixpkgs version:
```
$ nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion                                                                    
"16.09pre83147.df89584"
```

**Comment by FRidh** [comment]
It seems to give a fairly good reason as to why as well:
```
prompt_toolkit-1.0.0 not supported for interpreter python3.5m
```
This (in)direct dependency is not supported on Python 3.5. Note however that on master this has already been fixed.

**Comment by FRidh** [comment]
Furthermore, `pgcli` has `disabled = isPy35;`. According to their setup.py 3.5 is not supported.

**Comment by phanimahesh** [comment]
@FRidh Yes, the issue was to request disabling py3.5 package, I should have checked master, sorry.

**Comment by FRidh** [comment]
@phanimahesh I am looking into it now. There are some dependencies that have issues as well.

**Comment by FRidh** [comment]
fixed with d78a2b1ce9df168afd44f16ea07667d8e1e0732f and c632980a0231a2514e7acec3e8a15feb5a5373fa and f745e02510bb05aa199783155dc278949d65aa9b

**Comment by phanimahesh** [comment]
I'm new to Nix, I don't understand the fix fully, will look at it again later to see how this fixes it.

Thanks!
